### PR TITLE
Add all models

### PR DESCRIPTION
**Pull request by Mat1s**

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -13,4 +15,19 @@ def org_admin_list
     memberships.where(org_admin: true).map(&:user)
   end
 
+  aasm :column => 'state' do
```
Comment (pavlukov) [inline app/models/organization.rb]: fix hash rocket

File: db/migrate/20190207093429_drop_feedbacks.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class DropFeedbacks < ActiveRecord::Migration[5.2]
+  def change
+  	drop_table :feedbacks
```
Comment (StalemateInc) [inline db/migrate/20190207093429_drop_feedbacks.rb]: Fix identation

File: config/database.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
```
Comment (pavlukov) [inline config/database.yml]: delete this one

File: db/migrate/20190207094023_change_participations.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class ChangeParticipations < ActiveRecord::Migration[5.2]
+  def change
+  	add_column :participations, :await_check, :boolean
+  	add_reference :participations, :page
```
Comment (StalemateInc) [inline db/migrate/20190207094023_change_participations.rb]: Fix identation

**Pull request merged**